src/App.js

```diff
@@ -1,13 +1,15 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect } from 'react'
+import { Route } from 'wouter'
 import GlobalStyle from './theme/GlobalStyle'
 import ThemeProvider from './theme/ThemeProvider'
+import Home from './pages/Home'
+import Charcuterie from './pages/Charcuterie'
+import Faq from './pages/Faq'
```

Comment:
Might be good to have a `/pages/index.js` file which exports all of the pages so we can import pages like `import { Home, Charcuterie, Faq } from './pages'` or something